app/views/pages/instances/create.liquid

```diff
@@ -0,0 +1,11 @@
+---
+method: post
+slug: instances/create
```

Comment:
instead of `instances/create`, we recommend using `instances` to follow https://documentation.platformos.com/developer-guide/modules/platformos-modules#resourceful-route-naming-convention